std/format.d

```diff
@@ -5456,12 +5477,41 @@ private bool needToSwapEndianess(Char)(const ref FormatSpec!Char f)
     assert(stream.data == "C");
 }
 
+// Used to check format strings are compatible with argument types
+package static const checkFormatException(alias fmt, Args...) =
+{
+    try
+        .format(fmt, Args.init);
+    catch (Exception e)
+        return (e.msg is ctfpMessage) ? null : e;
```

Comment:
Hmm.  Are you sure the `is` will work?  Since `ctfpMessage` is declared as an enum, wouldn't it produce a new instance every time it's referenced?